.github/workflows/build-complete-samples.yml

```diff
@@ -849,3 +849,110 @@ jobs:
         run: |
           npm install
           npm run build --if-present
+
+  build-java-samples:
+    runs-on: windows-latest
+    #Add project path, name and version as below for any new additions
+    strategy:
+      matrix:
+        include:
+            - project_path: 'samples/bot-conversation/java'
+              name: 'bot-conversation'
```

Comment:
@Harikrishnan-MSFT - If you have already run this pipeline, please add the screenshot of executed pipeline.